packages/interface/src/components/more-menu-dropdown/style.scss

```diff
@@ -31,5 +31,5 @@
 }
 
 .components-popover.interface-more-menu-dropdown__content {
-	z-index: z-index(".components-popover.edit-widgets-more-menu__content");
+	z-index: z-index(".components-popover.interface-more-menu__content");
```

Comment:
This was wrong for some reason, so I'm changing it here to match the change in the _z-index.scss file.